CHANGES

```diff
@@ -1,7 +1,7 @@
 OpenVDB Version History
 =======================
 
-Version 8.0.0 - In development
+Version 8.0.0 - December 23, 2020
```

Comment:
Minor, probably should say 24th?

Reply:
Yes, I realise that. For the sake of not having to rebuild the docs, going to leave this a day out this time. :/